src/blocked.c

```diff
@@ -519,10 +519,9 @@ static void releaseBlockedEntry(client *c, dictEntry *de, int remove_key) {
         unblock_on_nokey_entry = dictFind(c->db->blocking_keys_unblock_on_nokey,key);
         /* it is not possible to have a client blocked on nokey with no matching entry */
         serverAssertWithInfo(c,key,unblock_on_nokey_entry != NULL);
-        dictIncrUnsignedIntegerVal(unblock_on_nokey_entry, -1);
+        if(!dictIncrUnsignedIntegerVal(unblock_on_nokey_entry, -1)) {
         /* in case the count is zero, we can delete the entry */
```

Comment:
indentation

Reply:
fixed in [c5104f2](https://github.com/redis/redis/pull/11012/commits/c5104f248e80ecb7f926f384317cf0ea18d28b4e)